src/common/framework/tree.ts

```diff
@@ -62,30 +62,37 @@ export interface TestTreeLeaf {
 
 export type TestTreeNode = TestSubtree | TestTreeLeaf;
 
+export type ExpandThroughLevel = 1 | 2 | 3;
```

Comment:
Could use a comment - do these levels mean suite, file, test ?

Do we plan to extend this further so you can pass any number and it'll expand params as well?
Oh I see - maybe that's what the `const expandThrough = Math.max(...` is for?